drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-client/src/test/java/org/drools/workbench/screens/dtablexls/client/handlers/NewDecisionTableXLSHandlerTest.java

```diff
@@ -237,6 +250,7 @@ public void testSuccessMultiByteProjectNameAndFileName() {
                                                         "default://" + encode( "ああ" ) + "/src/main/resources" );
 
         when( pkg.getPackageMainResourcesPath() ).thenReturn( resourcesPath );
+        when(uploadWidget.getFilenameSelectedToUpload()).thenReturn(fileName + ".xls");
```

Comment:
Small suggestion would you mind adding xlsx test as well?

Reply:
Done